SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -1177,7 +1178,7 @@ void CFVMFlowSolverBase<V, FlowRegime>::BC_Sym_Plane(CGeometry* geometry, CSolve
     for (auto iVar = 0u; iVar < nVar; iVar++) {
       if (iVar < iVel || iVar >= iVel + nDim) LinSysRes(iPoint, iVar) += residual.residual[iVar];
     }
-
+#endif
```

Comment:
@bigfooted do you remember what case motivated you to put these flux terms for continuity and energy?

Reply:
I believe it was the convergence of the compressible flow over a cylinder, or maybe the same setup but then switching to the flow over a section of a 3D sphere with 2 symmetry planes. But I ran these cases again just now and they converge fine with the current PR. The flow over a sphere is only a regression test for INC_NAVIER_STOKES, not for NAVIER_STOKES.

Reply:
We need at least part of the energy flux because of the work of pressure forces with moving grids, that is why the rotating NACA is not doing well.
I think the incompressible solver uses Cv == Cp (energy equal enthalpy) is that right? If so we don't need to do anything.

Reply:
Yes, we currently have only inc_ideal gas, where we use Cp=Cv+R
(There's also inc_ideal_gas_poly, but the only thing it does is make Cp a polynomial without any other modifications to the gas law)